projects/js-packages/charts/src/charts/sparkline/stories/index.api.mdx

```diff
@@ -0,0 +1,233 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="JS Packages/Charts Library/Charts/Sparkline/API Reference" />
+
+# Sparkline API Reference
+
+## Overview
+
+Sparkline is a preconfigured variant of [LineChart](?path=/docs/js-packages-charts-library-charts-line-chart--docs) optimized for compact, inline visualizations. It provides a simplified API while using LineChart's rendering engine internally.
+
+## Sparkline
+
+Main component for rendering minimal trend visualizations.
+
+**Props:**
+
+| Prop | Type | Default | Description |
+| ---- | ---- | ------- | ----------- |
+| `data` | `number[]` | - | **Required.** Array of numeric values to plot |
+| `width` | `number` | `100` | Width of the sparkline in pixels |
+| `height` | `number` | `40` | Height of the sparkline in pixels |
+| `size` | `number` | - | Size for responsive variant (equivalent to width for square charts) |
+| `color` | `string` | theme color | Color for the line stroke (hex or CSS color) |
+| `strokeWidth` | `number` | `1` | Line stroke width in pixels |
```

Comment:
```suggestion
| `strokeWidth` | `number` | `1.5` | Line stroke width in pixels |
```

Looks like the default strokeWidth is 1.5 set? - https://github.com/Automattic/jetpack/blob/fb3889bff089445abdfa7371f244cf08c16878c8/projects/js-packages/charts/src/charts/sparkline/sparkline.tsx#L12